### ISSUE TITLE
Get rid of StableRNG?

### ISSUE THREAD

**Issue by PallHaraldsson** [fipelle/MessyTimeSeriesOptim.jl#6]
Hi, I noticed you depend on StableRNG in both your packages, and use for tests (very valid, was made for that, and I thought not much else), but also in src/ for simulation. It may not be a big deal if it's not speed-critical, but I think using the default rng in non-test is going to be much faster (and higher quality). And then you can lose one (admittedly cheap, for startup-cost) dependency. [FYI: You can keep StableRNG for tests only, and for that purpose have a separate Project.toml file.]

FYI: The default rng changed in Julia 1.7, and it's much faster and multi-threaded. There's also other option, that may or may not be needed. If you have any questions please ask me. If this was intentional, just close the issue (your call if you explain).

**Comment by fipelle** [comment]
Hi Páll,

Thanks for taking the time to take a look at my time series packages. 

StableRNG is quite handy for testing and I am using it almost entirely for this very purpose. I have employed it in a few extra functions because I needed to have consistent results while porting the code from the old LTS version to the current one. These implementations are not slow or performance critical, but an extra kick is always welcome. Thus, I will definitely evaluate the default rng going forward!

Thanks for your feedback.
